community-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Polyhack Community Guidelines
+
+**[#polyhack](https://github.com/polyhack)** is a community of
+programmers based on the IRC freenode network. It is specifically a
+polyglot group concerned to discuss programming ideas over the long-term.
+
+Its membership is predominantly from the Australian east coast but
```

Comment:
Fixed. It was copied from polyhack.github.io so I submitted a fix for it there too.